### DISCUSSION TITLE
Can I autoplay the first animation at start?

### DISCUSSION THREAD

**Discussion by digitalPurist** [senshu/Sozi#554] [Q&A] [answered]
would be helpful i.e. to explain some things before the presentation starts.

thanks
digitalPurist

**Comment by aumouvantsillage** [comment, marked as answer]
Setting a timeout in the first frame will make the transition to the second frame automatic.
Would it be an acceptable solution for you?

**Reply by digitalPurist** [reply]
Ah, yes, working perfectly with timeout! 
thank you - your tool is very cool! fast and clean.
